Fluent/Physics.ipynb

```diff
@@ -6,70 +6,62 @@
+    "At this stage a detail undestanding about the velocity and pressure field within the flow over a stationary cylinder is obtained. In order to discuss the details of the flow field around a rotating cylinder one can add a clockwise vortex with strength $\\Gamma$ to the above-discussed flow. The combined streamfunction is the given by:\n",
     "\n",
-    "$$\\psi = Ur(1-\\frac{R^2}{r^2})sin\\theta + \\frac{\\land}{2\\pi}\\ln r$$\n",
+    "$$\\psi = U~r~(1-\\frac{R^2}{r^2})~sin\\theta + \\frac{\\Gamma}{2\\pi}~\\ln r$$\n",
     "\n",
-    "<img src=\"./Images/RotatingCylinder.jpg\" width=\"500\"> \n",
-    "*Figure 4. The addition of vortex of increasing strength to a uniform flow over a circular cylinder.*\n",
+    "Fig. 4 visualizes the streamline pattern for increasing vortex strength ($\\Gamma$). As shown in this figure the streamlines are more stacked at the top of the cylinder compared to the bottom. Therefore the velocity of the fluid is higher at the top compared to it's bottom. Consequently, according to the Bernoulli's equation, the pressure at the top of the cylinder is less than the bottom. This pressure difference between the top and bottom of the cylinder gives rise to a force in the perpendicular direction to the streamwise flow and is called the **Lift** force. This force increases with the strength of the vortex $\\Gamma$. Following the previous analysis one can show that the resultant lift force per unit span is given by:\n",
     "\n",
-    "Figure 4 shows streamline pattern for increasing vortex strength. As we can see that the streamlines are more stacked at the top of the cylinder than at the bottom. The velocity of the fluid is more at the top than at the bottom. Consequently, the pressure at the top is less than that at the bottom. This pressure difference gives rise to a force in the direction of decreasing pressure called the **Lift.** This force increases with the strength of the vortex. Following the previous analysis we can prove that the resultant lift force (per unit span) is given by\n",
+    "$$ F_L = \\rho~\\Gamma~U$$\n",
     "\n",
-    "$$ F_L = \\rho \\land U$$\n",
+    "<img src=\"./Images/RotatingCylinder.jpg\" width=\"500\"> \n",
+    "Fig. 4. Visualized streamlines around a cylinder modeled as a free vortex with strength of $\\Gamma$.\n",
     "\n",
     "In a viscous fluid, a cylinder spinning in an otherwise uniform flow can generate lift. A thin layer of fluid in the boundary layer is forced to spin with the cylinder because of viscous friction, and its effect on the surrounding flow is similar to that produced by the vortex on the flow over a non-spinning cylinder. The spin produces circulation around the cylinder, and the strength of this \"virtual\" vortex depends on the spin rate of the cylinder. The appearance of a side force on a spinning cylinder or a sphere is called the *Magnus Effect*, after the Swedish physicist Gustav Magnus who first described it clearly in 1853. \n",
     "\n",
```

Comment:
This is the conclusion part. State how all of the above analysis come together to describe the flow over a rotating cylinder. Also discuss how one will benefit from understanding the above theories to validate the CFD simulation results and how CFD simulation will bring into play the viscus effect to compliment this simple but yet valuable theoretical analysis.

Remove the reference to your paper in the end. We will have this on the front page of the case study and state that this repo. is based of this report of yours.